### PR TITLE
(EDIT): Edit Readme to reflect Testnet being live

### PR DESCRIPTION
**Pull request by xyizko**
# What ? 

Updating the readme regarding the testnet as as described [here](https://www.ithaca.xyz/updates/odyssey#how-do-i-try-out-odyssey)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,14 @@ Odyssey has 2 goals:
...
 
+### Odyssey Test Net
+
+> [!TIP]
+> [Testnet](https://www.ithaca.xyz/updates/odyssey#odyssey-chapter-1-is-live-on-testnet) is now live Sepolia and is built with Reth, the OP Stack, and [deployed on Conduit](https://app.conduit.xyz/published/view/odyssey).
+
 ### Odyssey Local Development
 
-Odyssey does not yet have a running testnet, but can be run locally for development and testing purposes. To do this, the binary can be run with the `--dev` flag, which will start the node with a development configuration.
+ Can be run locally for development and testing purposes. To do this, the binary can be run with the `--dev` flag, which will start the node with a development configuration.
```
Comment (Rjected) [inline README.md:37]:
```suggestion
Odyssey can be run locally for development and testing purposes. To do this, the binary can be run with the `--dev` flag, which will start the node with a development configuration.
```

File: README.md [hunk 2]
```diff
@@ -27,9 +27,14 @@ Odyssey has 2 goals:
 Odyssey's node extensions were chosen for their ability to enable applications that enhance the onchain user experience, and
 drastically reduce cost for existing applications that improve UX.
 
+### Odyssey Test Net
+
+> [!TIP]
+> [Testnet](https://www.ithaca.xyz/updates/odyssey#odyssey-chapter-1-is-live-on-testnet) is now live Sepolia and is built with Reth, the OP Stack, and [deployed on Conduit](https://app.conduit.xyz/published/view/odyssey).
+
```
Comment (Rjected) [inline README.md:34]:
```suggestion
### Odyssey Testnet

> [!TIP]
> [The Odyssey Testnet](https://www.ithaca.xyz/updates/odyssey#odyssey-chapter-1-is-live-on-testnet) is now live on Sepolia and is built with Reth, the OP Stack, and [deployed on Conduit](https://app.conduit.xyz/published/view/odyssey).

```

**Review by Rjected — CHANGES_REQUESTED** [review]
thx for the contribution! Just some grammar comments

**Review by mattsse — APPROVED** [review]

**Review by Rjected — APPROVED** [review]
thanks!

**Pull request merged**